logging/efk-logging-eventrouter.adoc

```diff
@@ -9,6 +9,11 @@ The Event Router communicates with the {product-title} and prints {product-title
 
 If Cluster Logging is deployed, you can view the {product-title} events in Kibana. 
 
+NOTE:
+====
+The Event Router is not supported for the Rsyslog log collector.
```

Comment:
should add that to the list of differences between fluentd and rsyslog

Reply:
Outdated